### PR TITLE
Fixed bug to incorrectly search installations for app name

### PR DESCRIPTION
**Pull request by ocke**
:v:

/cc @zendesk/vegemite

### Description
Currently the code looks for apps in the installation table which are not the same thing.

### References
* JIRA: https://zendesk.atlassian.net/browse/AF-XXX

### Risks
* [low] Apps are still not being found correctly.

### CODE REVIEW

File: lib/zendesk_apps_tools/deploy.rb [hunk 1]
```diff
@@ -66,22 +66,22 @@ def find_app_id(product_name = 'v2') # use the v2 endpoint if no product name is
       say_status 'Update', 'app ID is missing, searching...'
       app_name = get_value_from_stdin('Enter the name of the installed app:')
```
Comment (fvzzy) [inline lib/zendesk_apps_tools/deploy.rb]: Remove `installed`

File: lib/zendesk_apps_tools/deploy.rb [hunk 2]
```diff
@@ -66,22 +66,22 @@ def find_app_id(product_name = 'v2') # use the v2 endpoint if no product name is
...
 
-      response = cached_connection.get("/api/#{product_name}/apps/installations.json")
-      installations_json = json_or_die(response.body)
+      response = cached_connection.get("/api/#{product_name}/apps/owned.json")
+      owned_apps_json = json_or_die(response.body)
 
-      unless response.success? && installations_json.has_key?('installations')
+      unless response.success? && owned_apps_json.has_key?('apps')
         say_error_and_exit "Unable to retrieve installations. Please check your credentials and internet connection."
```
Comment (fvzzy) [inline lib/zendesk_apps_tools/deploy.rb]: Update to `Unable to retrieve apps. Please check your credentials and internet connection.`

**Comment by fvzzy** [comment]
Update https://github.com/zendesk/zendesk_apps_tools/blob/master/spec/lib/zendesk_apps_tools/deploy_spec.rb#L9

File: lib/zendesk_apps_tools/deploy.rb (hunk 2 above)
Comment (ocke) [reply lib/zendesk_apps_tools/deploy.rb]: Good eyes.

**Pull request merged**